.github/workflows/cln-plugin.yaml

```diff
@@ -0,0 +1,67 @@
+name: CI tests for CLN watchtower-plugin
+
+on: [push, pull_request]
+
+jobs:
+  cache-cln:
+    strategy:
```

Comment:
nit: don't think this line nor the `fail-fast` line are needed since this is not a matrix build.
Same for the ones in the next job.